avocado/etc/avocado/defaults.conf

```diff
@@ -0,0 +1,124 @@
+[datadir.paths]
```

Comment:
My suggestion is that this becomes a completely commented out configuration file, and have its name kept as `avocado.conf`.  This will ease migration, and can let us really implement builtin defaults (that don't depend on configuration files) and on our own selftests, check that configuration file settings are being applied more easily.

Reply:
Maybe I was not clear about this change.

The idea is that real soon defaults.conf will become defaults.py.